upgrade/UPGRADE-v9.2.0-dev.md

```diff
@@ -27,11 +27,43 @@ There you can find links to upgrade notes for other versions too.
         ```
     - for more information read our article [Icon function](https://docs.shopsys.com/en/9.1/frontend/icon-function/)
 
-**\[BC break\]** change entity extension subscriber class ([#2405](https://github.com/shopsys/shopsys/pull/2405))
+- **\[BC break\]** change entity extension subscriber class ([#2405](https://github.com/shopsys/shopsys/pull/2405))
     - see #project-base-diff to update your project 
     - package joschi127/doctrine-entity-override-bundle is no longer used
     - previously used subscriber `\Joschi127\DoctrineEntityOverrideBundle\EventListener\LoadORMMetadataSubscriber` was replaced with `\Shopsys\FrameworkBundle\Component\EntityExtension\EntityExtensionSubscriber`
         - if you have extended `LoadORMMetadataSubscriber`, you will need to extend `EntityExtensionSubscriber` instead and reimplement your changes on top of the new class
 
 - replace deprecated namespace `Doctrine\Common\Persistence\ObjectManager` with new `Doctrine\Persistence\ObjectManager`([#2407](https://github.com/shopsys/shopsys/pull/2407))
     - see #project-base-diff to update your project
+
+- **\[BC break\]** change used caches from doctrine/cache to Symfony cache ([#2413](https://github.com/shopsys/shopsys/pull/2413))
```

Comment:
hmm, I am not able to add a suggestion here :thinking: :disappointed: 

So:
```diff
- [#2413](https://github.com/shopsys/shopsys/pull/2413)
+ [#2412](https://github.com/shopsys/shopsys/pull/2412)
```

Reply:
👍 
fixed in 202a0913f